guardrails/validators.py

```diff
@@ -764,6 +766,50 @@ def validate(self, key: str, value: Any, schema: Union[Dict, List]) -> Dict:
         return schema
 
 
+class SimpleSqlDriver(SQLDriver):
```

Comment:
I think you forgot removing this after moving this to `sql_utils`?

Reply:
Thanks!